modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -285,11 +294,13 @@ In the following example, `{application-path-from-runtime-manager}` is set to `h
 curl -Lvk https://newapp.example-rtf.dev/ --resolve newapp.example-rtf.dev:443:192.168.64.14 
 ```
 
-== Step 4: Configure an External Load Balancer
+== Step 3: Configure an External Load Balancer
 After you enable inbound traffic, you must configure Runtime Fabric to route incoming traffic to each enabled application for clients to send requests to deployed applications. 
 
 You must configure an external load balancer to load balance HTTPS traffic between each controller VM on Runtime Fabric. Controller VMs are virtual machines dedicated to run the components that power Anypoint Runtime Fabric. Each controller VM runs a replica of the internal load balancer and is configured to listen on port 443. 
```

Comment:
> to balance HTTPS traffic

Is it always HTTPS traffic now that we have MVS?

Reply:
I added a clarification for this.